avalon/pipeline.py

```diff
@@ -720,6 +720,29 @@ def get_representation_context(representation):
     return context
 
 
+def get_work_directory():
+    """Return the current Avalon session work directory.
+
+    The work directory is the parsed work template of the project configuration
+    against the current Session dict.
+
+    Returns:
+        str: Path to work directory
+
+    """
+    project = io.find_one({"type": "project"})
+    template = project['config']['template']['work']
```

Comment:
Try not to intermix function with calls to the database. This would be better suited to having the template passed as an argument.

Reply:
Sure, so this would be more like a `format_template(template)` function?